src/Compilers/CSharp/Test/Semantic/FlowAnalysis/FlowDiagnosticTests.cs

```diff
@@ -1269,15 +1267,17 @@ public void AutoPropInitialization2()
     public S(int i) {}
 }";
 
-            var comp = CreateCompilation(text);
+            var comp = CreateCompilation(text, parseOptions: TestOptions.Regular9);
             comp.VerifyDiagnostics(
-    // (3,16): error CS0573: 'S': cannot have instance property or field initializers in structs
-    //     public int P { get; set; } = 1;
-    Diagnostic(ErrorCode.ERR_FieldInitializerInStruct, "P").WithArguments("S").WithLocation(3, 16),
-    // (5,20): error CS0573: 'S': cannot have instance property or field initializers in structs
-    //     public decimal R { get; } = 300;
-    Diagnostic(ErrorCode.ERR_FieldInitializerInStruct, "R").WithArguments("S").WithLocation(5, 20)
-);
+                // (3,16): error CS8652: The feature 'parameterless struct constructors' is currently in Preview and *unsupported*. To use Preview features, use the 'preview' language version.
+                //     public int P { get; set; } = 1;
+                Diagnostic(ErrorCode.ERR_FeatureInPreview, "P").WithArguments("parameterless struct constructors").WithLocation(3, 16),
+                // (5,20): error CS8652: The feature 'parameterless struct constructors' is currently in Preview and *unsupported*. To use Preview features, use the 'preview' language version.
+                //     public decimal R { get; } = 300;
+                Diagnostic(ErrorCode.ERR_FeatureInPreview, "R").WithArguments("parameterless struct constructors").WithLocation(5, 20));
+
+            comp = CreateCompilation(text, parseOptions: TestOptions.RegularPreview);
+            comp.VerifyDiagnostics();
```

Comment:
Could we add execution and VerifyIL verification?

Reply:
> Could we add execution and VerifyIL verification? 

Added in StructConstructorTests.